client/post-editor/media-modal/detail/detail-file-info.jsx

```diff
@@ -99,6 +100,51 @@ class EditorMediaModalDetailFileInfo extends React.Component {
 		);
 	};
 
+	renderUploadedTo = () => {
+		const parentTitle = this.getItemValue( 'parent_title' );
+		const parentLink = this.getItemValue( 'parent_link' );
+
+		if ( ! parentTitle || ! parentLink ) {
+			return;
+		}
+
+		return (
+			<tr>
+				<th>{ this.props.translate( 'Uploaded to' ) }</th>
+				<td>
+					<ExternalLink icon href={ parentLink }>
+						{ parentTitle }
+					</ExternalLink>
+				</td>
+			</tr>
+		);
+	};
+
+	renderAttachedTo = () => {
+		const attachedToDetails = this.getItemValue( 'attached_to' );
+
+		if ( ! attachedToDetails ) {
+			return;
+		}
+
+		return (
+			<tr>
+				<th>{ this.props.translate( 'Attached to' ) }</th>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **56** times:
`translate( 'Attached to', { context: 'column name'} )` *ES Score: 14*

Reply:
Looks like it's worth updating this translation to include the `context`, as it's also a column name.

Reply:
Will update these!